cards/src/main/resources/cards/custom/occultist/tgt/tgt/spell_bloody_blow.json

```diff
@@ -1,27 +1,27 @@
 {
   "name": "Bloody Blow",
-  "baseManaCost": 2,
+  "baseManaCost": 1,
   "type": "SPELL",
   "heroClass": "DARKGREEN",
   "rarity": "COMMON",
-  "description": "Take $4 damage. Deal $2 damage to all minions.",
+  "description": "Take $2 damage. Deal $2 damage to up to two random enemy minions.",
+   "condition": {
+    "class": "MinionCountCondition",
+    "value": 0,
+    "operation": "GREATER",
+    "targetPlayer": "OPPONENT"
+  },
   "targetSelection": "NONE",
-  "spell": {
-    "class": "MetaSpell",
-    "spells": [
-      {
-        "class": "DamageSpell",
-        "target": "ALL_MINIONS",
-        "value": 2
-      },
-      {
-        "class": "DamageSpell",
-        "target": "FRIENDLY_HERO",
-        "value": 4
-      }
-    ]
+   "spell": {

```

Comment:
This currently does not damage the friendly Champion. You should probably put back in the damage spell.